sharktank/sharktank/layers/paged_llama_attention_block.py

```diff
@@ -222,6 +232,7 @@ def forward(
             softcap=self.softcap,
             seq_lens=seq_lens,
             sliding_window=self.sliding_window,
+            sink=self.sink,
```

Comment:
How are the tests you linked to currently testing this path if its only now being implemented? The test also includes this TODO:

https://github.com/nod-ai/shark-ai/blob/7975496ca20463a03ff053d7b2c5379d2f537aab/sharktank/tests/layers/paged_llama_attention_block_test.py#L187-L187

Linking to https://github.com/nod-ai/shark-ai/issues/2156